install-jellyfin.ps1

```diff
@@ -0,0 +1,385 @@
+
+$ProgressBar                     = New-Object system.Windows.Forms.ProgressBar
+$ProgressBar.width               = 245
+$ProgressBar.height              = 30
+$ProgressBar.location            = New-Object System.Drawing.Point(70,5)
+
+$InstallLocationLabel            = New-Object system.Windows.Forms.Label
+$InstallLocationLabel.text       = "Install Location"
+$InstallLocationLabel.AutoSize   = $true
+$InstallLocationLabel.width      = 100
+$InstallLocationLabel.height     = 20
+$InstallLocationLabel.location   = New-Object System.Drawing.Point(5,50)
+$InstallLocationLabel.Font       = 'Microsoft Sans Serif,10'
+$GUIElementsCollection += $InstallLocationLabel
+
+$InstallLocationBox              = New-Object system.Windows.Forms.TextBox
+$InstallLocationBox.multiline    = $false
+$InstallLocationBox.width        = 205
+$InstallLocationBox.height       = 20
+$InstallLocationBox.location     = New-Object System.Drawing.Point(110,50)
+$InstallLocationBox.Text            = $Script:DefaultJellyFinInstallDirectory
+$InstallLocationBox.Font         = 'Microsoft Sans Serif,10'
+$GUIElementsCollection += $InstallLocationBox
+
+$InstallAsServiceCheck                       = New-Object system.Windows.Forms.CheckBox
+$InstallAsServiceCheck.text                  = "Install as Service?"
+$InstallAsServiceCheck.AutoSize              = $false
+$InstallAsServiceCheck.width                 = 140
+$InstallAsServiceCheck.height                = 20
+$InstallAsServiceCheck.location              = New-Object System.Drawing.Point(5,75)
+$InstallAsServiceCheck.Font                  = 'Microsoft Sans Serif,10'
+$GUIElementsCollection += $InstallAsServiceCheck
+
+$ServiceUserLabel            = New-Object system.Windows.Forms.Label
+$ServiceUserLabel.text       = "Service user"
+$ServiceUserLabel.AutoSize   = $true
+$ServiceUserLabel.width      = 100
+$ServiceUserLabel.height     = 20
+$ServiceUserLabel.location   = New-Object System.Drawing.Point(15,100)
+$ServiceUserLabel.Font       = 'Microsoft Sans Serif,10'
+$ServiceUserLabel.Visible    = $false
+$ServiceUserLabel.Enabled    = $false
+$GUIElementsCollection += $ServiceUserLabel
+
+$ServiceUserBox                  = New-Object system.Windows.Forms.ComboBox
+$ServiceUserBox.text             = "User for Service"
+$ServiceUserBox.width            = 195
+$ServiceUserBox.height           = 20
+@('Local System','Custom User') | ForEach-Object {[void] $ServiceUserBox.Items.Add($_)}
+$ServiceUserBox.location         = New-Object System.Drawing.Point(120,100)
+$ServiceUserBox.Font             = 'Microsoft Sans Serif,10'
+$ServiceUserBox.Visible          = $false
+$ServiceUserBox.Enabled          = $false
+$ServiceUserBox.DropDownStyle    = [System.Windows.Forms.ComboBoxStyle]::DropDownList
+$GUIElementsCollection += $ServiceUserBox
+
+$MigrateLibraryCheck                       = New-Object system.Windows.Forms.CheckBox
+$MigrateLibraryCheck.text                  = "Import Emby Library?"
```

Comment:
This is already a checkbox, why end in question mark?

Reply:
Fair enough, removed question marks